### PR TITLE
feat: remove hardSource

### PR DESCRIPTION
**Pull request by xiaohuoni**
##### Checklist

- [x] `npm test` passes
- [x] tests are included
- [x] documentation is changed or added
- [x] commit message follows commit guidelines

##### Description of change

[hardSource] is no longer valid

- close https://github.com/umijs/umi/issues/2213

### CODE REVIEW

**Comment by sorrycc** [comment]
依赖得删下。

**Comment by xiaohuoni** [comment]
还没改完

**Comment by xiaohuoni** [comment]
这个好了@sorrycc ，看看还要修改什么吗？

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -81,3 +81,6 @@ Deploy doc to [umijs.org](https://umijs.org/).
 ```bash
 $ y doc:deploy
 ```
+
+## Tips
+Node v10.*
```
Comment (sorrycc) [inline CONTRIBUTING.md]: Please use node@10, node@11 is not supported.

Comment (sorrycc) [reply CONTRIBUTING.md]: 标题下面加个空行。

**Pull request merged**